relay/messages/execution.go

```diff
@@ -42,7 +42,7 @@ type ExecutionResponse struct {
 	Bundle        string      `json:"bundle"`
 	Status        string      `json:"status"`
 	StatusMessage string      `json:"status_message"`
-	Template      string      `json:"template"`
+	Template      interface{} `json:"template"`
```

Comment:
Nope. It's required to hit the correct function head on the cog side of things that uses a default template. With a string value here the default value in the json is an empty string instead of `null`.